CRM/MembersOnlyEvent/Form/MembersOnlyEventTab.php

```diff
@@ -284,25 +288,18 @@ public function postProcess() {
     $params = $this->exportValues();
     $params['event_id'] = $this->_id;
 
-    $eventSetToMembersOnly = $params['event_access_type'] === 'members_only';
-    $eventSetToGroupsOnly = $params['event_access_type'] === 'groups_only';
-    $isTabEnabled = $eventSetToMembersOnly || $eventSetToGroupsOnly;
-
-    // Skip if both are true. We don't want to handle this edge case.
-    if ($eventSetToMembersOnly && $eventSetToGroupsOnly) {
-      return FALSE;
-    }
+    $isTabEnabled = (bool) $params['event_access_type'];
```

Comment:
This seems to be incorrect (casting value to boolean) as  I understand the `event_access_type ` value would be either 1,2 or 3 as you defined in BAO class.

Reply:
If the user submitted the form without selecting any option then its value is empty string and in that case the relevant record in the table `membersonlyevent` will be deleted.

Reply:
Created the commit [WHF-308: Use empty function to check if the tab is enabled](https://github.com/compucorp/uk.co.compucorp.membersonlyevent/pull/30/commits/e8cf0a9e23ac3548217698bdfe1686c8e3fdc8ad) which results in better code readability.